src/libraries/System.Net.Security/src/System/Net/Security/SslStream.Implementation.cs

```diff
@@ -440,7 +440,7 @@ private int CheckEnqueueRead(Memory<byte> buffer)
                 _lockReadState = LockPendingRead;
             }
             // Need to exit from lock before waiting.
-            lazyResult.InternalWaitForCompletion();
+            lazyResult!.InternalWaitForCompletion();
```

Comment:
I might be missing something, but I don't see how this can be non-null.

Reply:
What was a conclusion here? Looks tricky if it is not a bug.

Reply:
We never get here.  It's a known bug. @wfurt has been working on it.

Reply:
Tracking in #32920

Reply:
https://github.com/dotnet/runtime/pull/32925